docs/content/2.composables/9.useDirectusFields.md

```diff
@@ -0,0 +1,173 @@
+# `useDirectusFields`
+
+---
+
+> Check out the Directus [Fields](https://docs.directus.io/reference/system/fields.html) documentation.
+
+### `getFields`
+
+Get all fields in a specific collection
+
+- **Arguments:**
+
+  - data: [`DirectusFieldsRequest`](https://github.com/Intevel/nuxt-directus/blob/main/src/runtime/types/index.d.ts)
+    - collection?: `string`
+
+- **Returns:** `Promise<DirectusField[]>`
+
+```vue [pages/example.vue]
+<script setup lang="ts">
+const { getFields } = useDirectusFields();
+
+const fetchFields = async () => {
+  try {
+    const fields = await getFields({
```

Comment:
Can you add a generic type here to the docs? Like in the playground usage.

Reply:
@Intevel done!